dashboards/heroku/postgres.json

```diff
@@ -26,7 +26,7 @@
             "tags": [
               {
                 "key": "addon",
-                "value": "*"
+                "value": "postgresql-*"
```

Comment:
That rename (adding the prefix) could break existing graphs and triggers, right? What's our plan for that?

Reply:
Existing graphs are already migrated :)